### PR TITLE
fix the modelValue Booleanish

### PR DESCRIPTION
**Pull request by xvaara**
As discussed in discord.

### CODE REVIEW

File: packages/bootstrap-vue-next/src/components/BPopover.vue [hunk 1]
```diff
@@ -165,17 +165,16 @@ interface BPopoverEmits {
 
 const emit = defineEmits<BPopoverEmits>()
 
-const showState = ref(props.modelValue)
+const modelValueBoolean = useBooleanish(toRef(props, 'modelValue'))
+const showState = ref(modelValueBoolean.value)
 watch(showState, (value) => {
-  emit('update:modelValue', !!value)
+  emit('update:modelValue', value)
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BPopover.vue:171]: It doesn't let me make a suggestion to the full but you should change this to
```ts
watchEffect(() => {
  emit('update:modelValue', !!showState.value) 
}
```

File: packages/bootstrap-vue-next/src/components/BPopover.vue [hunk 2]
```diff
@@ -165,17 +165,16 @@ interface BPopoverEmits {
...
+})
+
+watch(modelValueBoolean, () => {
```
Comment (VividLemon) [inline packages/bootstrap-vue-next/src/components/BPopover.vue:174]: watchEffect

**Review by VividLemon — CHANGES_REQUESTED** [review]

**Review by VividLemon — APPROVED** [review]

File: packages/bootstrap-vue-next/src/components/BPopover.vue (hunk 1 above)
Comment (xvaara) [reply packages/bootstrap-vue-next/src/components/BPopover.vue]: Isn't !! unnecessary here?

Comment (VividLemon) [reply packages/bootstrap-vue-next/src/components/BPopover.vue]: Yeah, it can be omitted.

**Pull request merged**